pkgs/tools/misc/ethminer/default.nix

```diff
@@ -31,6 +32,21 @@ stdenv.mkDerivation rec {
       fetchSubmodules = true;
     };
 
+  patches = [
```

Comment:
This whole chunk only really adds `fetchpatch` with a fix for cuda 11. The rest is moving `postPatch` and `patches` from the bottom closer to `src`